client/my-sites/plugins/plugin-autoupdate-toggle/index.jsx

```diff
@@ -133,21 +133,16 @@ export class PluginAutoUpdateToggle extends Component {
 	}
 
 	render() {
-		const { inProgress, site, plugin, translate, disabled } = this.props;
+		const { inProgress, site, plugin, disabled } = this.props;
 		if ( ! site.jetpack ) {
 			return null;
 		}
 
 		const getDisabledInfo = this.getDisabledInfo();
-		const label = translate( 'Autoupdates', {
-			comment:
-				'this goes next to an icon that displays if the plugin has "autoupdates", both enabled and disabled',
-		} );
 
 		return (
 			<PluginAction
 				disabled={ disabled }
-				label={ label }
```

Comment:
Can we use aria-label instead of completely removing any text that would help with accessibility?

Reply:
Now this keeps the label attribute and we pass as aria-label inside the component:  36aeab24044da59454aabc46a6f36caa963c0809